common/main/root.xml

```diff
@@ -5369,6 +5369,8 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 	</typographicNames>
 	<personNames>
 		<nameOrder nameLocales="und" order="surnameFirst"/>
+		<initialPattern type="initial">{0}.</initialPattern>
+		<initialPattern type="initialSequence">{0}{1}</initialPattern>
```

Comment:
I think more languages use spaces between initials than not, so maybe this should also be space separated.

Reply:
@macchiati OK, I added spaces (I only had it without spaces because that is the way it was in the agenda to-dos), and have squashed. Those are the only changes, needs re-approval to merge, thanks.